aces/Loris
# [Statistics / Behavioural Statistics] fix 500 error

## Brief summary of changes

This PR address the issues related to the error found in  #6573 but without including the project and site restrictions in #6649 that need to be improved.

#### Testing instructions (if applicable)

1. Go to `Reports->Statistics->Behavioural Statistics`
2. Select different projects on the `Projects ComboBox `and click on `Submit Query`
3. The 500 error should be solved (the user should be able to query for any project without issue).

#### Link(s) to related issue(s)

* Resolves #6573
* Same solution that provided on #6649 but without including the project and site restrictions in that one.
